extend.php

```diff
@@ -66,6 +69,9 @@
     (new Extend\ApiSerializer(GroupSerializer::class))
         ->attributes(Api\AddGroupAttributes::class),
 
+    (new Extend\ApiSerializer(ForumSerializer::class))
+        ->attributes(Api\AddForumAttributes::class),
```

Comment:
Why are we serializing to the qr logo url to the forum?

Reply:
https://github.com/flarum/framework/blob/e731fb61896fe96f67f49734eba7dd25a1fb96e5/framework/core/js/src/admin/components/UploadImageButton.js#L18C13-L18C13


In `UploadImageButton` needs to show a preview in admin so it concatenates the setting key with the `Url` (see the link above)